src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -303,6 +304,10 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('web_link.xml');
         }
 
+        $loggerDefinition = $container->register(LoggerInterface::class, Logger::class);
+        $loggerDefinition->setPublic(false);
+        $container->setAlias('logger', LoggerInterface::class);
```

Comment:
the alias also needs to be private.
for other services, we create the "noun" as a real definition, and the "fqcn" as the alias
for consistency, shouldn't we do the same here also?